src/cpp/server/server_cc.cc

```diff
@@ -77,14 +77,8 @@ class DefaultGlobalCallbacks final : public Server::GlobalCallbacks {
   void PostSynchronousRequest(ServerContext* context) override {}
 };
 
-std::shared_ptr<Server::GlobalCallbacks> g_callbacks = nullptr;
-gpr_once g_once_init_callbacks = GPR_ONCE_INIT;
-
-void InitGlobalCallbacks() {
-  if (!g_callbacks) {
-    g_callbacks.reset(new DefaultGlobalCallbacks());
-  }
-}
+DefaultGlobalCallbacks* g_default_callbacks = new DefaultGlobalCallbacks();
+Server::GlobalCallbacks* g_callbacks = g_default_callbacks;
```

Comment:
Nit: Please rename g_callbacks to global_callbacks and g_default_callbacks to global_default_callbacks

Reply:
Thanks for your review. For consistency with client_context.cc, should I keep the g_ prefix or use global_ here?

https://github.com/grpc/grpc/blob/master/src/cpp/client/client_context.cc#L45-L48

Reply:
I would prefer changing it to global_ as I'm not a fan of single character abbreviations :) I understand your approach of keeping it consistent with the existing code, but I'd push for changing it here.

Reply:
Done.